docs/source/hardware/low_priority_faqs.md

```diff
@@ -0,0 +1,208 @@
+(low-priority-faqs)=
+
+# Low Priority Partition FAQs
+
+## Partition Information
+
+::::{dropdown} 1. What is the `lowpriority` partition and why is it important?
+`lowpriority` is a new partition on Discovery that allows the research community to use **private resources** on the cluster when they are idle. This new partition has hardware not otherwise available to the general research community and, in time, could double the resources available to NURC users. This is a common practice in HPC clusters to optimize the use of idle private resources that consume power and cooling. 
+
+More information about the low priority partition and examples on how to use it can be 
+found {ref}`low-priority`.
+::::
+
+::::{dropdown} 2. What is a "low priority" job?
+A job submitted to the `lowpriority` partition that comes from a user **who is NOT** a member of the private partition on which the job is scheduled to run.
+::::
+
+::::{dropdown} 3. What is a “high priority” job?
+A job submitted to a private partition by a user **who is** a member of that private partition, on which a low priority job is running.
+::::
+
+::::{dropdown} 4. How does SLURM handle low priority jobs?
+In SLURM, jobs are submitted to partitions. SLURM is configured such that jobs submitted to the `lowpriority`partition are considered for execution on private partitions only when the requested job resources are idle. 
```

Comment:
This whole section looks good! I just had one question: Does the node state have to be "idle" for a job to be run? Or is it that the node has to be not in use by a member of the private partition for outside members jobs to be run?

For example, say a node is being used by low priority by a user who isn't part of the private partition, the node is no longer idle, can another job (from a none private partition member) be run on the same node (if allowable with the requested resources)?

Reply:
From my experience, idle is just available resources and not the actual IDLE NODELIST reason. From what I remember, multiple users can all use a single node as long as there are available resources too. Even people who can normally run on the private hardware can coexist with lowpriority users but if the private hardware group requires more resources than what is available that is when jobs start getting cancelled for the low priority users. 

Greg would definitely be able to answer this in more definitely phrasing but this is what I recall from using a lowpriority partition previously but I was on the other end and didn't have complete visibility.